docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Spelling mistake in "VALMOP"</p><p> </p><p>Do you mean "bottom-up" approach?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Yes. I will fix both.</p>